v2/datastream-to-spanner/terraform/samples/mysql-end-to-end/README.md

```diff
@@ -67,27 +70,26 @@ Following permissions are required -
 
 **Note**: Add the `roles/viewer` role as well to the service account.
 
-> **_Note on IAM:_** 
+> **_Note on IAM:_**
 >
 > 1. For ease of use, this sample automatically adds the
 > required
 > roles to the service account used for running the migration. In order to
-> do this, we need the `resourcemanager.projects.setIamPolicy` permission. If
-> granting
+> do this, we need the `roles/iam.securityAdmin` role. If granting
```

Comment:
This section is at a permission revel, you should talk about the specific permission and not the role here.